tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -30,7 +30,7 @@ Request with non-existing log type
     ...    fragments={"c8y_LogfileRequest":{"dateFrom":"${start_timestamp}","dateTo":"${end_timestamp}","logFile":"example1","searchText":"first","maximumLines":10}}
     Operation Should Be FAILED
     ...    ${operation}
-    ...    failure_reason=.*No such file or directory for log type: example1
+    ...    failure_reason=.*No logs found for log type example1
```

Comment:
I've created (and will likely work on soon if no one objects) https://github.com/thin-edge/thin-edge.io/issues/2487 for this

Reply:
No objections. But, in the meantime you can also use single quotes or back quotes to get around that problem.